src/Tool/Arrays.php

```diff
@@ -0,0 +1,35 @@
+<?php declare(strict_types=1);
```

Comment:
Do we intend to use this class in other plugins too in the future?  I mean, why we've placed it to the core, not to the Fuzzy plugin?

Reply:
It offers commonly used features that can be useful for other plugins as well. Currently, it's used by two plugins, including the Query Suggestion plugin, which is still being finalized.